Update the Readme files (#22)
DevExpress-Examples/devextreme-cascading-selectboxes: React/README.md

# React

> Take a look at the following file to see the code of this example: 

Reviewer: ```suggestion
The following file contains the code of this example:
```
